modules/cluster-logging-rn-5.5.3.adoc

```diff
@@ -0,0 +1,36 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-3_{context}"]
+= Logging 5.5.3
+This release includes link:https://access.redhat.com/errata/RHBA-2022:6858[OpenShift Logging Bug Fix Release 5.5.3].
+
+[id="openshift-logging-5-5-3-bug-fixes_{context}"]
+== Bug fixes
+* Before this update log entries that had structured messages included the original message field which made the entry larger. This update removes the message field for structured logs to reduce the increased size. (link:https://issues.redhat.com/browse/LOG-2759[LOG-2759])
+
+* Before this update, the collector configuration excluded logs from `collector`, `default-log-store`, and `visualization`  pods, but was unable to exclude logs archived in a `.gz` file. With this update, archived logs stored as `.gz` files of `collector`, `default-log-store`, and `visualization` pods are also excluded. (link:https://issues.redhat.com/browse/LOG-2844[LOG-2844])
+
+* Before this update, when requests to an unavailable pod was sent through the gateway, no alert would fire to warn of the disruption. With this update, the individual alerts will fire if the gateway has issues completing a write or read request respectively. (link:https://issues.redhat.com/browse/LOG-2884[LOG-2884])
```

Comment:
`fire` might be too colloquial of a verb. 

also, verb-tense agreement with `requests` and `was` 

what do you think of this rephrase?

`Before this update, when requests to an unavailable pod were sent through the gateway, no alert would warn of the disruption.`